std/random.d

```diff
@@ -2008,6 +2008,11 @@ if (isRandomAccessRange!Range && hasLength!Range && isUniformRNG!RandomGen)
     return range[uniform(size_t(0), $, urng)];
 }
 
+auto ref choice(Range)(auto ref Range range)
```

Comment:
Should be `/// ditto` - this tells the Ddoc engine to visually group the overloads.